modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js

```diff
@@ -170,7 +170,7 @@ class Sidebar extends Component {
 					...field,
 					defaultLanguageId,
 					editingLanguageId,
-					readOnly: this.isFieldReadOnly(field),
+					readOnly: false,
```

Comment:
@natocesarrego This is actually a JSON property of a field. Therefore, it is necessary to assign value because it can be used elsewhere.

Reply:
@natocesarrego This is actually a JSON property of a field and has no default value. Therefore, it is necessary to assign value because it can be used elsewhere.

Reply:
Great! Thanks for the clarification. 🙂